docs/installation.md

```diff
@@ -0,0 +1,79 @@
+## Installation
+This guide uses minishift version v1.33.0+ as the local Kubernetes cluster
+and quay.io for the public registry. We also test on minikube version v0.35+.
+`kubectl` is used but can be substituted* with `oc`.
+
+### Supported versions
```

Comment:
Do we want to have a table here instead? 

| Version | Workload | Supported|
|-----------|--------------|---------------|
| 3.11      | Couchbase | |
| 3.11      | UPerf | x |

For example.

Reply:
ah, think I miss understood this section.